transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollServerSocketChannelConfig.java

```diff
@@ -145,6 +150,14 @@ public EpollServerSocketChannelConfig setMessageSizeEstimator(MessageSizeEstimat
     }
 
     /**
+     * Set the {@code TCP_MD5SIG} option on the socket. See {@code linux/tcp.h} for more details.
+     */
```

Comment:
Add the same note as here:
https://github.com/netty/netty/pull/4173/files#diff-f157068a80d067588aa75aed3471003eR329